src/models/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./azureProvider";
```

Comment:
Why the exports?

Reply:
To make model imports easy. I can remove these